nifi-assembly/NOTICE

```diff
@@ -1364,6 +1364,8 @@ The following binary components are provided under the MIT License.  See project
 
   (MIT License) EWS Java API (com.microsoft.ews-java-api:ews-java-api:2.0 - https://github.com/OfficeDev/ews-java-api)
   (MIT License) libffi (libffi-3.2.1 - http://sourceware.org/libffi/)
+  (MIT License) Kitteh IRC Client Library (kitteh client-lib 3.0.3 - http://kicl.kitteh.org/)
+  (MIT License) 'mbassador' (mbassador 1.2.4 - https://github.com/bennidi/mbassador)
```

Comment:
lines 1365-1368 should be deleted.  MIT licensed dependencies get referenced in the LICENSE and do not end up in the NOTICE.